js/components/notificationBar.js

```diff
@@ -115,6 +116,14 @@ class NotificationBar extends ImmutableComponent {
   }
 }
 
+class NotificationBarCaret extends ImmutableComponent {
```

Comment:
I am not sure if we need component for two divs. What I would suggest is to create divs where we now use this component and move styles into notificationStyle file and use it from there.
cc @cezaraugusto